CMakeLists.txt

```diff
@@ -666,24 +666,38 @@ endif()
 
 # Configuration file YAML format support
 if(FLB_CONFIG_YAML)
-# Requies libyaml support
-check_c_source_compiles("
-  #include <yaml.h>
-  int main() {
-    yaml_parser_t parser;
-    return 0;
-  }" FLB_HAVE_LIBYAML)
+  find_package(PkgConfig)
+  # libyaml's corresponding pkg-config file is yaml-0.1.pc.
+  # We should search it first.
+  pkg_check_modules(LIBYAML QUIET yaml-0.1)
+
+  if (LIBYAML_FOUND)
+    # For if(FLB_HAVE_LIBYAML) clause on CMakeList.txt.
+    set(FLB_HAVE_LIBYAML 1)
+    FLB_DEFINITION(FLB_HAVE_LIBYAML)
+    # For non-standard libyaml installation paths such as homebrew bottled libyaml.
+    include_directories(${LIBYAML_INCLUDEDIR})
+    link_directories(${LIBYAML_LIBRARY_DIRS})
+  else()
+    # Requies libyaml support
```

Comment:
typo
```suggestion
    # Requires libyaml support
```

Reply:
Oh, I'd overlooked about it. Fixed in https://github.com/fluent/fluent-bit/pull/5641/commits/bdc3b2cddfe0df95351be12c342684af6ecb5e11